### DISCUSSION TITLE
From Address is always zero address

### DISCUSSION THREAD

**Discussion by semiPlanet** [0xPolygon/polygon-edge#680] [General]
Hi, I am confused why the from address is always zero in my blockchain. I am running 2 chains in one it is ok but in another the from address is always zero.
`{"jsonrpc":"2.0","id":1,"result":{"nonce":"0x0","gasPrice":"0x12a05f200","gas":"0x5208","to":"0x7735562229A3d765D468E8B424C3bb0fEdb068E7","value":"0xde057346fb07000","input":"0x","v":"0x647","r":"0x1ac31b7214360ef9827f4a7af9f8ecda6907f2931f148ba4afa2e0b6dc2ec395","s":"0x65671d0ea6d6387934a89b40d7f492124c128d45b4ddd1a71cd6a9f7ddba115c","hash":"0x6b6e24664f8b175e6de52dc7fe7dbc173a507fbb0d84bab9383bdd5b605eab1e","from":"0x0000000000000000000000000000000000000000","blockHash":"0x9cbd48129e8f836d8bd1d6d46190ad663793c5c049473f42b182f875e6606674","blockNumber":"0x71f","transactionIndex":"0x0"}}`

You can also see this in the given link: https://nxchainscan.com/tx/0x6b6e24664f8b175e6de52dc7fe7dbc173a507fbb0d84bab9383bdd5b605eab1e
https://nxchainscan.com/tx/0xaa188456206a0343ac30e7476683254b253dfe33c25bb5da13ba48015b7abd42

Please help me to resolve this issue.
Thank you.

**Comment by laviniat1996** [comment]
Hi @semiPlanet,
Thank you for opening this discussion.

We need more information in order to investigate. I would suggest you to open an issue for this. Please use this [template](https://docs.polygon.technology/docs/edge/community/report-bug) to report the problem, it gives us more insight.

**Comment by semiPlanet** [comment]
Hi, Thank you for your reply. I have opened an issue as you have suggested. You can view the issue through given link
https://github.com/0xPolygon/polygon-edge/issues/684

Thanks
